modules/ob-disabling-shipwright-custom-resource.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-a-shipwright-build-resource_{context}"]
+= Disabling ShipwrightBuild resource
```

Comment:
I don't like this phrasing - as a customer I would imagine this has very little meaning. The customer is _disabling Shipwright Builds_.